_posts/news/2020-11-16-announcing-scalajs-1.3.1.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: Announcing Scala.js 1.3.1
+category: news
+tags: [releases]
+permalink: /news/2020/11/16/announcing-scalajs-1.3.1/
+---
+
+
+We are pleased to announce the release of Scala.js 1.3.1!
+
+This is primarily a bugfix release, including fixes for three regressions in 1.3.0 ([#4252](https://github.com/scala-js/scala-js/issues/4252), [#4268](https://github.com/scala-js/scala-js/issues/4268) and [#4271](https://github.com/scala-js/scala-js/issues/4271)).
+It also contains some improvements to the JDK support.
+
+More importantly, this release fixes the performance issues of the ECMAScript 2015 mode (the default) on Firefox.
```

Comment:
Re-reading this, it feels contradictory with the previous paragraph: If this is primarily a bugfix release, then why is it *more* important that it fixes a performance issue on FF?

IMO the performance fix should go first, as it is likely to affect the largest fraction of users unexpectedly (folks that care about regressions or new features are likely actively looking for them in the release notes).

Reply:
(re-re-reading this, I realize that a performance fix can also be considered a bugfix... :shrug: )